Purchases/Public/Purchases.swift

```diff
@@ -944,10 +945,11 @@ public extension Purchases {
      *
      * - Note: This may force your users to enter the App Store password so should only be performed on request of
      * the user. Typically with a button in settings or near your purchase UI. Use
-     * ``Purchases/syncPurchases(completionBlock:)`` if you need to restore transactions programmatically.
+     * ``Purchases/syncPurchases(completion:)`` if you need to restore transactions programmatically.
```

Comment:
The label is `maybeCompletion` in `syncPurchases`

Reply:
Updated param `maybeCompletion` in favor of `completion`